lib/fog/storage/openstack/requests/delete_container.rb

```diff
@@ -1,6 +1,12 @@
 module Fog
   module Storage
     class OpenStack
+      class Mock
+        def delete_container(name)
+          fail "Mock Not Implemented (#delete_container) in: #{__FILE__}:#{__LINE__}"
```

Comment:
Gotcha. Maybe you might want to add this to fog-core instead so all providers will have this feature?

Also, does this work with all active maintained ruby versions?